plugins/badges/src/routes.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { createRouteRef } from '@backstage/core';
+
+export const badgeRouteRef = createRouteRef({
```

Comment:
should make this one specific (so we can add more badge types in the future), and add a type that describes the needed parameters.

```suggestion
import { EntityName } from '@backstage/catalog-model';
export const entityBadgeRouteRef = createRouteRef<EntityName>({
```

Reply:
Oh, cool. I think I accidently removed this routeRef, as I didn't see it used, which probably was my bad, as I have hardcoded routes elsewhere instead... will look at it after the weekend.

Reply:
I'm not using a routeRef any longer, as I don't declare any routes, I suppose that's fine?